### DISCUSSION TITLE
Changing ID

### DISCUSSION THREAD

**Discussion by AEnnemoser** [rustdesk/rustdesk#3757] [Q&A]
Hi there,
so we installed a few PCs with the same Image and for some reason they now all have the same Rustdesk ID. We tried changing it manually on the PCs and we were able to change it through the rustdesk.toml file until you restart the PC. After restarting the same old ID will return and you can now only connect to one of these PCs.
Is there some way to change them locally or does it have to be changed on the server?

Alex

**Comment by korneevigor** [comment]
Такая же ситуация, что делать?

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/pro/
